CHANGES.md

```diff
@@ -9,6 +9,8 @@
 - Enable MSAA by default with 4 samples. To turn MSAA off set `scene.msaaSamples = 1` [#12158](https://github.com/CesiumGS/cesium/pull/12158)
 - Made the `time` parameter optional for `Property`, using `JulianDate.now()` as default. [#12099](https://github.com/CesiumGS/cesium/pull/12099)
 - Exposes `ScreenSpaceCameraController.zoomFactor` to allow adjusting the zoom factor (speed). [#9145](https://github.com/CesiumGS/cesium/pull/9145)
+- Expose the `tonemapper` property of `PostProcessStageCollection` to allow changing the tonemap used when HDR is turned on. This defaults to the [PBR Neutral Tonemap from Khronos](https://github.com/KhronosGroup/ToneMapping/tree/main/PBR_Neutral) [#12160](https://github.com/CesiumGS/cesium/pull/12160)
+  - As part of this change we switched the default tonemap for models, atmosphere and globe when _not_ using HDR to the new PBR Neutral tonemap from the ACES tonemap to improve visuals.
```

Comment:
I'd move this to a separate item under "Breaking Changes" since it will affect everyone:

`Switched the default (non-HDR) tonemapping for models, atmosphere, and globe from ACES to [PBR Neutral Tonemap from Khronos](https://github.com/KhronosGroup/ToneMapping/tree/main/PBR_Neutral). The widens the gamut of possible colors, and provides more consistent color appearances across renderers. [#12160](https://github.com/CesiumGS/cesium/pull/12160)`

Reply:
I'd also add an item to breaking changes that mentions the HDR default has changed tonemappers from ACES to PBR Neutral in "Breaking Changes", and explain how to set it back to the previous default.

It's really important to be clear and specific in `CHANGES.md` so users know what to expect when they update.